device.mk

```diff
@@ -16,6 +16,10 @@ PRODUCT_COPY_FILES += \
     $(LOCAL_PATH)/rootfs/init.poplar.rc:root/init.poplar.rc \
     $(LOCAL_PATH)/rootfs/init.poplar.usb.rc:root/init.poplar.usb.rc \
 
+ifeq ($(TARGET_TEE_IS_OPTEE), true)
```

Comment:
Seem this is not an Android predefined variable? Who and when will set this to true?

Reply:
No this isn't an Android predefined variable. I think Victor requested optee be integrated in this way, so it maybe easier to upstream to AOSP, and it is easy to switch the TEE to trusty for example. It is designed to be set when you compile android e.g. 'make -j12 TARGET_TEE_IS_OPTEE=true'. This is also similar to how HiKey is integrating optee currently.